packages/editor-ui/src/components/mixins/nodeHelpers.ts

```diff
@@ -389,5 +392,24 @@ export const nodeHelpers = mixins(
 				}
 				return undefined;
 			},
+			getFixedNodesList(workflowNodes: INode[]) {
```

Comment:
sorry I meant canvas helpers, not node helpers, because this function does not require any state. goal of canvas helpers is to slowly break apart  NodeView.vue into non-stateful helper functions so that we can easily refactor it eventually.